### DISCUSSION TITLE
I want to search for files with the .php extension

### DISCUSSION THREAD

**Discussion by sherproger** [community/community#115599] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

For example, I want to search for files with the .php extension and sort the search results by recent changes, how do I do it?

**Comment by mike-clark-8192** [comment, marked as answer]
In the search bar you can specify `path:*.php` (glob) or `path:/\.php$/` (regex). But I am not sure how to do the sorting.

**Reply by look** [reply]
Great answer by @mike-clark-8192. You can also do `language:PHP` which may include related extensions. There is no way to sort by recently changed.
